domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/GjeldendeVedtaksdata.kt

```diff
@@ -25,10 +25,22 @@ data class GjeldendeVedtaksdata(
     val grunnlagsdataOgVilkårsvurderinger: GrunnlagsdataOgVilkårsvurderinger.Revurdering
 
     private val tidslinje: Tidslinje<VedtakSomKanRevurderes.VedtakPåTidslinje> = vedtakListe
-        .lagTidslinje(periode)
+        .lagTidslinje(periodeForTidslinje)
 
     private val vedtakPåTidslinje: List<VedtakSomKanRevurderes.VedtakPåTidslinje> = tidslinje.tidslinje
 
+    /* Periode som representerer tidligest og senest dato på vedtaken i tidslinjen */
+    val periode: Periode
+        get() =
+            vedtakPåTidslinje
+                .map { it.periode }
+                .let { perioder ->
+                    Periode.create(
```

Comment:
Det finnes en List<Periode>.minAndMaxOf. @jacob-meidell vil dette være trygt nok? Det gir rammene til tidslinjen, men tidslinjen trenger ikke være kontinuerlig. Javadocen burde kanskje tydeliggjøre dette.

Reply:
Hmm.. det kommer an på. Det egentlige problemet er vel at konsumenter av denne "perioden" har lagt til grunn en del forutsetninger som ikke nødvendigvis vil være sann for alltid. Når man tidligere kun brukte fraOgMed, var man i hvert fall garantert at man var begrenset til vedtak som eksisterte på saken (dette i seg selv gjør det ikke trygt å bruke perioden, ref hull etc, og vil måtte suppleres med ekstra sjekker for å avgjøre om det er trygt).